deepchecks/checks/distribution/train_test_label_drift.py

```diff
@@ -125,15 +125,14 @@ def condition(result: Dict) -> ConditionResult:
                          (drift_score > max_allowed_earth_movers_score and method == "Earth Mover's Distance")
 
             if method == 'PSI' and has_failed:
-                return_str = f'Label has PSI over {max_allowed_psi_score} - Drift score is {drift_score:.2f}'
+                return_str = f'Found label PSI above threshold: {drift_score:.2f}'
                 return ConditionResult(False, return_str)
             elif method == "Earth Mover's Distance" and has_failed:
-                return_str = f'Label has Earth Mover\'s Distance over {max_allowed_earth_movers_score} - ' \
-                             f'Drift score is {drift_score:.2f}'
+                return_str = f'Found label Earth Mover\'s Distance above threshold: {drift_score:.2f}'
```

Comment:
```suggestion
                return_str = f'Label Earth Mover\'s Distance above threshold: {drift_score:.2f}'
```

Reply:
So you are fine with the "found" everywhere else except label drift?